src/lib/components/workflow/workflows-summary-configurable-table/table-body-cell.svelte

```diff
@@ -102,6 +102,14 @@
       })}
     {:else if label === 'History Length'}
       {parseInt(workflow.historyEvents, 10) > 0 ? workflow.historyEvents : ''}
+    {:else if label === 'Scheduled By ID'}
+      {workflow.searchAttributes?.indexedFields?.TemporalScheduledById ?? ''}
+    {:else if label === 'Scheduled Start Time'}
+      {@const content =
+        workflow.searchAttributes?.indexedFields?.TemporalScheduledStartTime}
+      {content && typeof content === 'string'
+        ? formatDate(content, $timeFormat, { relative: $relativeTime })
```

Comment:
Are these decoded already? I know they don't need to go to the codec server but they are payloads so need to be base64 decoded

Reply:
Yep, they are decoded when we get them back from the server: https://github.com/temporalio/ui/blob/main/src/lib/models/workflow-execution.ts#L39